src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1115,7 +1164,7 @@ private void RemoveWorseMembers<TMember>(ArrayBuilder<MemberResolutionResult<TMe
                     Debug.Assert(!results[i].Result.IsValid || worse[i] != unknown);
                     if (worse[i] == worseThanSomething || worse[i] == notBetterThanEverything)
```

Comment:
Due to the assertion  `Assert(countOfNotBestCandidates == 0 || !haveBestCandidate)`  the condition `worse[i] == notBetterThanEverything` is unreachable. Should I remove it?